tests/D2L.CodeStyle.Analyzers.Test/Specs/OnlyVisibleToAnalyzer.cs

```diff
@@ -9,7 +9,7 @@
 namespace D2L.CodeStyle.Annotations.Contract {
 
 	[AttributeUsage(
-		validOn: AttributeTargets.Method | AttributeTargets.Property,
+		validOn: AttributeTargets.Interface | AttributeTargets.Method | AttributeTargets.Property,

```

Comment:
I'm noticing the lack of the `AttributeTargets.Constructor` here, which is on the main attribute. I'm not sure if that should be here?

Reply:
Do we have tests with constructors? If not I guess I can see why it works. Maybe we should have tests for that... 🤷 Maybe not a big deal